### PR TITLE
added rlp

### PR DESCRIPTION
**Pull request by step21**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dopplershift** [comment]
Nothing obviously wrong, but clearly CI is unhappy.

**Comment by step21** [comment]
It's missing a package for running, I will check whether I can fix this with other packages or comment it out temporarily.

**Comment by step21** [comment]
@conda-forge-admin, please rerun ci

**Comment by step21** [comment]
(it might have to wait until eth-utils is available until this works)

**Comment by step21** [comment]
@conda-forge-admin, please rerun ci

File: recipes/rlp/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (dopplershift) [inline recipes/rlp/LICENSE:1]: It looks like the `LICENSE` is already included in the tarball, so you don't need to add it here. You should be able to leave the `license_file` below unchanged, however.

Comment (step21) [reply recipes/rlp/LICENSE:1]: I thought that I still have to add it, to be safe. But ok.

Comment (dopplershift) [reply recipes/rlp/LICENSE]: No, because the build will error out if it can't find the file specified by `license_file`.

**Pull request merged**